tests/dist/chats.spec.dev.js

```diff
@@ -336,35 +346,36 @@ describe('chat resolvers', function () {
   test('remove group chat', function _callee7() {
     var removeGroupChatResponse, removeGroupChatData;
     return regeneratorRuntime.async(function _callee7$(_context7) {
-      while (1) {
+      const condition = 1;
+      while (condition) {
         switch ((_context7.prev = _context7.next)) {
           case 0:
             _context7.next = 2;
             return regeneratorRuntime.awrap(
-            axios.post(
-              URL,
-              {
-                query: '\n    mutation{\n      removeGroupChat(chatId:"'.concat(
-                  createdGroupChatId,
-                  '") {\n        _id\n    }\n    }\n            '
-                ),
-              },
-              {
-                headers: {
-                  Authorization: 'Bearer '.concat(token),
+              axios.post(
+                URL,
+                {
+                  query: '\n    mutation{\n      removeGroupChat(chatId:"'.concat(
+                    createdGroupChatId,
+                    '") {\n        _id\n    }\n    }\n            '
+                  ),
```

Comment:
It would be ideal to use a template literal here to improve the readability of this query.